data/json/professions.json

```diff
@@ -6441,5 +6441,40 @@
       "male": [ "boxer_shorts" ],
       "female": [ "panties", "bra" ]
     }
+  },
+  {
+    "type": "profession",
+    "id": "explosive_ordinance_disposal_on_duty",
+    "name": "Explosive Ordinance Disposal Technician",
+    "description": "Ever since you were a child you were fascinated with explosives and all things that go boom. As soon as you turned 18 you joined the army to become an EOD technician. When your base was overrun, you find yourself alone in the middle of nowhere with nothing but your precious training, gear and cherished love of explosives",
+    "points": 5,
+    "proficiencies": [ "prof_elec_circuits", "prof_intro_chemistry", "prof_athlete_expert", "prof_gun_cleaning" ],
+    "skills": [
+      { "level": 4, "name": "melee" },
+      { "level": 4, "name": "gun" },
+      { "level": 4, "name": "rifle" },
+      { "level": 3, "name": "pistol" },
+      { "level": 3, "name": "dodge" },
+      { "level": 3, "name": "firstaid" },
+      { "level": 3, "name": "throw" },
+      { "level": 3, "name": "survival" }
+    ],
```

Comment:
```suggestion
    "proficiencies": [ "prof_elec_circuits", "prof_intro_chemistry", "prof_athlete_expert", "prof_gun_cleaning", "prof_spotting", "prof_traps", "prof_disarming", "prof_trapsetting" ],
    "skills": [
      { "level": 6, "name": "traps" },
      { "level": 4, "name": "melee" },
      { "level": 4, "name": "gun" },
      { "level": 4, "name": "rifle" },
      { "level": 3, "name": "pistol" },
      { "level": 3, "name": "dodge" },
      { "level": 3, "name": "firstaid" },
      { "level": 3, "name": "throw" },
      { "level": 3, "name": "survival" }
    ],
```

Reply:
Will add that ASAP completely forgot about trap skill!